bin/main.rb

```diff
@@ -1,2 +1,55 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
+
+# Welcome to Tic Tac Toe Game
+puts 'Welcome to Tic Tac Toe Game'
+
+# Display game board
+
+board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+puts '-------------'
+puts "| #{board[0]} | #{board[1]} | #{board[2]} |"
+puts '-------------'
+puts "| #{board[3]} | #{board[4]} | #{board[5]} |"
+puts '-------------'
+puts "| #{board[6]} | #{board[7]} | #{board[8]} |"
+puts '-------------'
+
+# Ask to Player 1 put X
+
+puts 'Player 1, place X'
+player1 = gets.chomp.to_i
+board[player1 - 1] = 'X'
+
+# Display the X inside the board in player 1 choice
+
+puts 'Your move is displayed on the board'
+puts '-------------'
+puts "| #{board[0]} | #{board[1]} | #{board[2]} |"
+puts '-------------'
+puts "| #{board[3]} | #{board[4]} | #{board[5]} |"
+puts '-------------'
+puts "| #{board[6]} | #{board[7]} | #{board[8]} |"
+puts '-------------'
+
+# Ask to Player 2 put O
+
+puts 'Player 2, place O'
+player2 = gets.chomp.to_i
+board[player2 - 1] = 'O'
+
+# Display the X inside the board in player 1 choice
+
+puts 'Your move is displayed on the board'
+puts '-------------'
+puts "| #{board[0]} | #{board[1]} | #{board[2]} |"
+puts '-------------'
+puts "| #{board[3]} | #{board[4]} | #{board[5]} |"
+puts '-------------'
+puts "| #{board[6]} | #{board[7]} | #{board[8]} |"
+puts '-------------'
+
+# Repeat user inputs until there is a winner
+
```

Comment:
- [ ] [REQUIRED] A loop is needed in this game (recommended is while but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png))
Please place a loop here which terminates after a winning or draw move. 
**NB:** Not everything has to work but the milestone should show that you have thought of every scenario in the game flow. The changes listed above are all that you are left with. 😃